src/commands/remote_signer_setup.py

```diff
@@ -65,6 +75,32 @@
     prompt='Enter comma separated list of API endpoints for execution nodes',
     help='Comma separated list of API endpoints for execution nodes.',
 )
+@click.option(
+    '--update-db',
+    is_flag=True,
+    help='Whether to update the database with keystores data for web3signer.',
+    default=False,
+)
+@click.option(
+    '--db-url',
+    help='The database connection address.' "ex. 'postgresql://username:pass@hostname/dbname'",
+    prompt=False,
+    required=False,
+)
+@click.option(
+    '--encryption-key',
+    help='The key for encrypting database record. '
+    'If you are upload new keystores use the same encryption key.',
+    required=False,
+    prompt=False,
+)
+@click.option(
```

Comment:
Consider removing this flag. It's not clear here that it is only related to the DB functionality. Another option is to make sure that the whole command get use of it (e.g. if there are more prompts - silence them as well.)